src/services/sessions.js

```diff
@@ -48,7 +46,7 @@ export async function getAllSessions() {
 
 export async function saveSession(params) {
   const { sessionConfig, description } = params;
-  const endpoint = `${endpoints.api.graphql}`;
+  const endpoint = `${window.endpoints.api.graphql}`;
```

Comment:
And here, and several additional places in the file. It's not incorrect; but it'd look neater without the backtick substitution.